stories/features/abort.fmf

```diff
@@ -0,0 +1,21 @@
+title: Abort test execution
+story:
+    As a tester I want to abort current, and all subsequent,
+    test executions using the restraint command during test
+    execution.
```

Comment:
@pfdaly 

I would not mention restraint here, actually `tmt-abort` is not a restraint command ...

```suggestion
    As a tester I want to abort current, and all subsequent,
    test executions using a shell command during test
    execution.
```

WDYT?

Reply:
That makes sense. I've updated that in my most recent commit.